src/commands/env/api.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { readFile } from 'node:fs/promises';
+import got, { Headers, Method } from 'got';
+import * as chalk from 'chalk';
+import * as ProxyAgent from 'proxy-agent';
+import { getProxyForUrl } from 'proxy-from-env';
+import { SfCommand, Flags } from '@salesforce/sf-plugins-core';
+import { SfError, Messages, Org } from '@salesforce/core';
+import { CliUx } from '@oclif/core';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.load('@cristiand391/sf-plugin-api', 'env.api', [
+  'summary',
+  'description',
+  'examples',
+  'flags.target-org.summary',
+  'flags.method.summary',
+  'flags.include.summary',
+  'flags.header.summary',
+  'flags.body.summary',
+  'errors.invalid-http-header',
+]);
+
+export default class EnvApi extends SfCommand<string> {
+  public static summary = messages.getMessage('summary');
+  public static description = messages.getMessage('description');
+  public static examples = messages.getMessages('examples');
+  public static enableJsonFlag = false;
+  public static flags = {
+    'target-org': Flags.requiredOrg({
+      summary: messages.getMessage('flags.target-org.summary'),
+      char: 'o',
+    }),
+    include: Flags.boolean({
+      summary: messages.getMessage('flags.include.summary'),
+      default: false,
+    }),
+    method: Flags.enum<Method>({
+      options: ['GET', 'POST', 'PUT', 'PATCH', 'HEAD', 'DELETE', 'OPTIONS', 'TRACE'],
```

Comment:
`got` accepts lowercase HTTP methods but I decided to only support the uppercase ones only, the `--help` output for the command adds the possible options so having a bunch seems like a lot of noise.